ete4/smartview/renderer/faces.py

```diff
@@ -546,7 +548,12 @@ def draw(self, drawer):
         self._check_own_variables()
 
         circ_drawer = drawer.TYPE == 'circ'
-        style = {'fill': self.color, 'opacity': self.opacity}
+        style = {
+            'fill': self.color, 
+            'opacity': self.opacity,
+            'stroke': self.stroke_color,
+            'stroke_width': self.stroke_width
```

Comment:
I think the last line should be:

```py
            'stroke-width': self.stroke_width
```

Otherwise the stroke-width is ignored.

Reply:
Thanks! I corrected it at the last commit